ts/components/NewRemindEmailValidationOverlay.tsx

```diff
@@ -46,34 +47,32 @@ import { IOToast } from "./Toast";
 import { LightModalContextInterface } from "./ui/LightModal";
 import { withLightModalContext } from "./helpers/withLightModalContext";
 import BaseScreenComponent from "./screens/BaseScreenComponent";
+import { CountdownProvider, useCountdown } from "./countdown/CountdownProvider";
 
-const emailSentTimeout = 10000 as Millisecond; // 10 seconds
+const emailSentTimeout = 10000 as Millisecond; // 60 seconds
 const profilePolling = 5000 as Millisecond; // 5 seconds
 
 const EMPTY_EMAIL = "";
 const VALIDATION_ILLUSTRATION_WIDTH: IOPictogramSizeScale = 80;
 
 type OwnProp = {
   isOnboarding?: boolean;
+  sendEmailAtFirstRender?: boolean;
 };
 
 type Props = LightModalContextInterface & OwnProp;
 
-const NewRemindEmailValidationOverlay = (props: Props) => {
-  const { isOnboarding, hideModal } = props;
+const NewRemindEmailValidationOverlayInner = (props: Props) => {
```

Comment:
We could change this name `NewRemindEmailValidationOverlayInner` with a better one. What do you think?

Reply:
I agree with renaming, but I don't like this proposal because it doesn't seem useful to me and doesn't seem to add value. We could think to change naming with EmailValidationOverlay, or a more "talky" name

Reply:
This `NewRemindEmailValidationOverlayInner` was not a proposal, it was indeed e temporary name I used for fast developing. I'm asking to you what better name to use instead of this ;)

Reply:
ah ok, sorry I did not understand, anyway i change the naming using NewRemindEmailValidationOverlayComponent